snow/networking/timeout/manager_test.go

```diff
@@ -33,17 +32,13 @@ func TestManagerFire(t *testing.T) {
 	)
 	require.NoError(t, err)
 	go manager.Dispatch()
-
-	wg := sync.WaitGroup{}
-	wg.Add(1)
+	defer manager.Stop()
 
 	manager.RegisterRequest(
 		ids.NodeID{},
 		ids.ID{},
 		true,
 		ids.RequestID{},
-		wg.Done,
+		nil,
 	)
-
-	wg.Wait()
```

Comment:
This seems to be deleting the actual test

Reply:
Indeed I misunderstood the point of the test. Fixed.